packages/components/src/border-control/types.ts

```diff
@@ -109,6 +110,10 @@ export type DropdownProps = ColorProps &
 		 * values for its popover controls.
 		 */
 		border?: Border;
+		/**
+		 * Whether a border style can be set, based on the border sanitization settings.
+		 */
+		isStyleSettable: boolean;
```

Comment:
Just an addition to an internal subcomponent prop — will not affect public-facing types.